oneview_redfish_toolkit/blueprints/network_adapter.py

```diff
@@ -65,7 +65,7 @@ def get_network_adapter(uuid, device_id):
 
         if (device_id_validation - 1) < 0 or (device_id_validation - 1) >= \
             len(server_hardware["portMap"]["deviceSlots"]):
-            logging.error(server_hardware)
+            logging.exception(server_hardware)
```

Comment:
Here!

Reply:
remove, this was inserted for debugging and forgot to remove